Meta/ladybird.sh

```diff
@@ -99,8 +100,17 @@ cmd_with_target() {
         LADYBIRD_SOURCE_DIR="$(get_top_dir)"
         export LADYBIRD_SOURCE_DIR
     fi
-    BUILD_DIR="$LADYBIRD_SOURCE_DIR/Build/ladybird"
-    CMAKE_ARGS+=("-DCMAKE_INSTALL_PREFIX=$LADYBIRD_SOURCE_DIR/Build/ladybird-install")
+
+    declare -A BUILD_DIRECTORIES
+
+    # Note: Keep in sync with buildDir defaults in CMakePresets.json
+    BUILD_DIRECTORIES["default"]="$LADYBIRD_SOURCE_DIR/Build/ladybird"
```

Comment:
Hmm. that kind of advanced use case goes out of scope of the script here I think.

That can easily (for some definition of easy) be done by talking to CMake and Ninja directly like so:

```
cmake --preset default -B Build/feature-xyz
ninja -C Build/feature-xyz
```

For fixing the behavior of BUILD_PRESET on ladybird.sh I think we should go with this patch, but I can create an issue to make sure the docs are clear on how to control your build directly, using the configs in CMakePresets.json as a template rather than the truth.

Reply:
https://github.com/LadybirdBrowser/ladybird/issues/583